src/librustc_typeck/check/wfcheck.rs

```diff
@@ -301,16 +313,34 @@ fn check_item_fn<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>, item: &hir::Item) {
     })
 }
 
+pub fn check_item_sized<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>, id: NodeId) {
```

Comment:
Apparently this should be done in `librustc_typeck/check/mod.rs` using `require_type_is_sized` and not in wfcheck.
So this leaves wfcheck with `check_item_type` only.

Reply:
Should it be done in `CheckItemTypesVisitor.visit_item` in `librustc_typeck/check/mod.rs` for example?

Reply:
Probably in `fn check_item_type`